### PR TITLE
fix(example): hpa of gke example doesn't work

### LINKED ISSUE #224: HPA of Google GKE example doesn't work
**What is the bug?**

HPA failed because we didn't specify the resource for `gitSync` sidecar

```console
Conditions:
  Type           Status  Reason                   Message
  ----           ------  ------                   -------
  AbleToScale    True    SucceededGetScale        the HPA controller was able to get the target's current scale
  ScalingActive  False   FailedGetResourceMetric  the HPA was unable to compute the replica count: missing request for cpu
Events:
  Type     Reason                        Age                   From                       Message
  ----     ------                        ----                  ----                       -------
  Warning  FailedComputeMetricsReplicas  20m (x12 over 23m)    horizontal-pod-autoscaler  invalid metrics (1 invalid out of 1), first error is: failed to get cpu utilization: missing request for cpu
  Warning  FailedGetResourceMetric       3m21s (x80 over 23m)  horizontal-pod-autoscaler  missing request for cpu
```

**What are your Helm values?**

<https://github.com/airflow-helm/charts/blob/68c17dbce1/charts/airflow/examples/google-gke/custom-values.yaml>

**What is your Kubernetes Version?**:

```console
Client Version: version.Info{Major:"1", Minor:"19", GitVersion:"v1.19.7", GitCommit:"1dd5338295409edcfff11505e7bb246f0d325d15", GitTreeState:"clean", BuildDate:"2021-01-13T13:23:52Z", GoVersion:"go1.15.5", Compiler:"gc", Platform:"darwin/amd64"}
Server Version: version.Info{Major:"1", Minor:"18+", GitVersion:"v1.18.9-eks-d1db3c", GitCommit:"d1db3c46e55f95d6a7d3e5578689371318f95ff9", GitTreeState:"clean", BuildDate:"2020-10-20T22:18:07Z", GoVersion:"go1.13.15", Compiler:"gc", Platform:"linux/amd64"}
```

**What is your Helm version?**:

```console
version.BuildInfo{Version:"v3.4.0", GitCommit:"7090a89efc8a18f3d8178bf47d2462450349a004", GitTreeState:"dirty", GoVersion:"go1.15.3"}
```

### PR DESCRIPTION
**Pull request by david30907d**
**What issues does your PR fix?**

- fixes https://github.com/airflow-helm/charts/issues/224

**What does your PR do?**

copy an [old settings](https://github.com/airflow-helm/charts/blob/0f40ffe1c1/charts/airflow/values.yaml#L1018-L1025) in Airflow 1.10 era to V2

## Checklist

- [x] Commits are [signed](https://github.com/airflow-helm/charts/tree/main/CONTRIBUTING.md#sign-your-work)
- [x] Commits are [squashed](https://github.com/airflow-helm/charts/tree/main/CONTRIBUTING.md#squash-commits) (if appropriate)
- [ ] Chart [version bumped](https://github.com/airflow-helm/charts/tree/main/CONTRIBUTING.md#versioning)
- [x] Chart [documentation updated](https://github.com/airflow-helm/charts/tree/main/CONTRIBUTING.md#documentation)
- [ ] Passes [linting](https://github.com/airflow-helm/charts/tree/main/CONTRIBUTING.md#linting)

### CODE REVIEW

File: charts/airflow/examples/google-gke/custom-values.yaml [hunk 1]
```diff
@@ -314,7 +314,14 @@ dags:
     ## the key in `dags.gitSync.sshSecret` with your ssh-key file
     ##
     sshSecretKey: id_rsa
```
Comment (thesuperzapper) [inline charts/airflow/examples/google-gke/custom-values.yaml:316]: @david30907d can you add a newline here

File: charts/airflow/examples/google-gke/custom-values.yaml [hunk 2]
```diff
@@ -314,7 +314,14 @@ dags:
...
-
+    ## NOTE:
+    ## - when `workers.autoscaling` is true, YOU MUST SPECIFY a resource request
+    ##
+    ## EXAMPLE:
+    resources:
+      requests:
+        cpu: "50m"
+        memory: "64Mi"
```
Comment (thesuperzapper) [inline charts/airflow/examples/google-gke/custom-values.yaml:324]: @david30907d can you add a newline here

**Review by thesuperzapper — CHANGES_REQUESTED** [review]
Good catch!

If you can just add some newlines it looks good to go.

File: charts/airflow/examples/google-gke/custom-values.yaml (hunk 1 above)
Comment (david30907d) [reply charts/airflow/examples/google-gke/custom-values.yaml:316]: done~

File: charts/airflow/examples/google-gke/custom-values.yaml (hunk 2 above)
Comment (david30907d) [reply charts/airflow/examples/google-gke/custom-values.yaml:325]: done 👍

File: charts/airflow/examples/google-gke/custom-values.yaml [hunk 3]
```diff
@@ -315,6 +315,15 @@ dags:
     ##
     sshSecretKey: id_rsa
 
+    ## NOTE:
+    ## - when `workers.autoscaling` is true, YOU MUST SPECIFY a resource request
+    ##
+    ## EXAMPLE:
+    resources:
```
Comment (thesuperzapper) [inline charts/airflow/examples/google-gke/custom-values.yaml:322]: @david30907d Can you format this docstring in the standard way:

```
## resource requests/limits for the git-sync container
##
## WARNING:
## - you MUST SPECIFY a resource request for gitSync if using `workers.autoscaling`
##
resources:
  ...
```

Comment (david30907d) [reply charts/airflow/examples/google-gke/custom-values.yaml]: done 😄

**Review by thesuperzapper — APPROVED** [review]
Thanks @david30907d!

**Pull request merged**